src/main/java/chronosacaria/mcda/registry/ArmorsRegistry.java

```diff
@@ -0,0 +1,136 @@
+package chronosacaria.mcda.registry;
+
+import chronosacaria.mcda.Mcda;
+import chronosacaria.mcda.items.ArmorSets;
+import chronosacaria.mcda.items.ArmorSetItem;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.item.Item;
+import net.minecraft.util.registry.Registry;
+
+import java.util.*;
+
+import static net.minecraft.entity.EquipmentSlot.*;
+
+public class ArmorsRegistry {
+    // (set, slot) -> item
+    public static EnumMap<ArmorSets, EnumMap<EquipmentSlot, Item>> armorItems = new EnumMap<>(ArmorSets.class);
+
+    protected static String armorID(ArmorSets set, EquipmentSlot slot) {
+        String slotId;
+        switch (slot) {
+            case HEAD:
+                slotId = "helmet";
+                break;
+            case CHEST:
+                slotId = "chestplate";
+                break;
+            case LEGS:
+                slotId = "leggings";
+                break;
+            case FEET:
+                slotId = "boots";
+                break;
+            default:
+                throw new IllegalArgumentException("armor with non-armor equipment slot");
+        }
+
+        return set.getSetName() + "_" + slotId;
```

Comment:
Luckily, the item ids follow this simple pattern - I hope that they all actually do, because if not, players will lose some equipment when loading this new version.